packages/shared/src/modules3rdParty/react-native-image-colors/index.ts

```
@@ -0,0 +1,34 @@
+import { getColors } from 'react-native-image-colors';
+
+import platformEnv from '../../platformEnv';
+
+import type { ImageColorsResult } from 'react-native-image-colors';
+
+const parseHexColor = (hexColor: string) => {
+  const r = parseInt(hexColor.slice(1, 3), 16);
```

Biome

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)